calyx-py/calyx/builder.py

```diff
@@ -268,53 +277,64 @@ def seq_mem_d1(
             name, ast.Stdlib.seq_mem_d1(bitwidth, len, idx_size), is_external, is_ref
         )
 
-    def add(self, name: str, size: int, signed=False) -> CellBuilder:
-        """Generate a StdAdd cell."""
+    def binary_op_helper(
+        self,
+        operation: str,
+        size: int,
+        name=None,
+        signed=False,
```

Comment:
Consider including type annotations:

```suggestion
        name: Option[str] = None,
        signed: bool = False,
```